runtime/etc/Makefile.exe

```diff
@@ -24,11 +24,17 @@ include $(CHPL_MAKE_HOME)/runtime/etc/Makefile.include
 
 all: $(TMPBINNAME)
 
+blank :=
+define newline
+
+$(blank)
+endef
```

Comment:
What is the role of `$(blank)` in the above?  I found myself at https://stackoverflow.com/posts/29651474/revisions which doesn't seem to use/require it?

I'd suggest adding a comment above `define newline` saying something like "This defines a newline character that can be used to push commands onto their own distinct line"?